docs/CloudPermissions.md

```diff
@@ -0,0 +1,56 @@
+# Restricting cloud privileges for deploying Scalar DL
+
+This guide explains how to restrict privileges for deploying Scalar DL on Azure.
+
+In general from security perspective, it is better to assign restricted privileges to users not to make them able to do operations that they are not supposed to do.
+
+## Azure
+
+The following JSON is a custom role that allow users to manage resources for Scalar DL. This is sufficient but not necessary.
```

Comment:
for Scalar DL?
or for k8s?
With the permissions, scalar dl is the only one that can be deployed? or other containers can be deployed?
if the latter, then it sounds misleading.

Reply:
For managing cloud resources for Scalar DL, changed the sentence in c601da6